app/views/users/edit.html.erb

```diff
@@ -0,0 +1,25 @@
+<h1>Update</h1>
+
+<%= form_for(@user) do |f| %>
+
+  <%= f.label :first_name %>
+  <%= f.text_field :first_name %>
+
+  <%= f.label :last_name %>
+  <%= f.text_field :last_name %>
+
+  <%= f.label :username %>
+  <%= f.text_field :username %>
+
+  <%= f.label :email %>
+  <%= f.email_field :email %>
+
+  <%= f.label :password %>
```

Comment:
@CjMoore I think it might be good to ask them for their password before they make any changes.